packages/eslint-plugin/lib/config/jsx-a11y.ts

```diff
@@ -1,42 +1,36 @@
 import type { Linter } from "eslint";
-import { sourceFiles } from "../utils/patterns";
+import { concat } from "eslint-flat-config-utils";
+import jsxA11yPlugin from "eslint-plugin-jsx-a11y";
+import { sourceFiles } from "../utils/patterns.ts";
 
-const config: Linter.Config = {
-    overrides: [
-        {
-            files: sourceFiles,
-            plugins: ["jsx-a11y"],
-            parserOptions: {
-                ecmaFeatures: {
-                    jsx: true
-                }
-            },
-            extends: [
-                "plugin:jsx-a11y/recommended"
-            ],
-            rules: {
-                // There is a really good article that describes the issues with autoFocus and why it should be avoided:
-                // https://brucelawson.co.uk/2009/the-accessibility-of-html-5-autofocus/
-                // However, this issue is with screen readers and not with keyboard navigation.
-                // In Workleap, we use autoFocus in a lot of places to improve the user experience.
-                // Therefore, we are disabling this rule.
-                "jsx-a11y/no-autofocus": "off",
+const config: Linter.FlatConfig[] = await concat(
+    {
+        name: "Workleap/JsxA11y",
+        plugins: { "jsx-a11y": jsxA11yPlugin },
```

Comment:
I am confused by this one, I thought plugins were not a thing anymore?

Reply:
I feel like it either we define a plugin that takes care of merging everything or we merge everything ourself, not both?

Reply:
At this point, plugins are just used to define custom rule behavior. The one Workleap custom rule requires the `workleapPlugin` to be provided as a plugin. If anyone wanted to just use that rule, they would have to provide to provide the `workleapPlugin` as a plugin.